### PR TITLE
Added the dashboard layout

### PR DESCRIPTION
**Pull request by luigimorel**
## Description
Please include a summary of the change and relevant motivation and context. 
I have added the desktop layout of the dashboard which will house the child components.

All pull requests should be related to open issues. Indicate the issue(s) bellow and remove this line:
Fixes (#33)

## How Has This Been Tested?
Please describe in detail how you tested your changes.
Include details of your testing environment, and the tests you ran to see how your change affects other areas of the code etc.

## Screenshots (if applicable, else remove this line / section)
Screenshot (309)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by MutegekiHenry** [comment]
@morelmiles is this PR work in progress? you could add a label for that.

**Comment by luigimorel** [comment]
I sent it as a draft. You can't access it?


---
Luigi Morel.

Front-end developer and UI/UX designer.

Tel: +256 704 916 696 (WhatsApp)

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,14 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
-
+import Sidebar from './components/Sidebar/Sidebar';
```
Comment (MutegekiHenry) [inline src/App.js:3]: @morelmiles this file is going to break our current codebase for the features in place

Comment (MutegekiHenry) [reply src/App.js:3]: the way around it is to first rebase with develop branch, then also import this component in the routes.js file

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,14 @@
...
+import ChartsDisplay from './components/ChartsDisplay/ChartsDisplay';
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
```
Comment (MutegekiHenry) [inline src/App.js:8]: this doesnt look like our current App.js on develop

**Review by MutegekiHenry — CHANGES_REQUESTED** [review]

**Comment by MutegekiHenry** [comment]
@morelmiles you have conflicts with the base branch .. and its because you didn't checkout from the updated version of develop on your forked branch... we could have a call to resolve these together

File: src/App.js (hunk 1 above)
Comment (onabright) [inline src/App.js:3]: @morelmiles what naming convention are we using for the component folders/names? camelCase, PascalCase or some other case? I suggest we pick and stick to one.

**Review by onabright — CHANGES_REQUESTED** [review]
Work on the changes that @MutegekiHenry suggested.

File: src/App.js (hunk 1 above)
Comment (MutegekiHenry) [reply src/App.js:3]: I agree

Comment (MutegekiHenry) [reply src/App.js:3]: Personally was using PascalCase

**Comment by MutegekiHenry** [comment]
@morelmiles this PR has stayed unattended to for a while, let me pull and address the comments, we can always catch up on the changes over a call

**Review by pearlonzo — APPROVED** [review]

**Comment by MutegekiHenry** [comment]
@pearlonzo we don't just approve a PR that is labelled work in progress and shows conflicts.

**Comment by MutegekiHenry** [comment]
I have closed it since it was way too behind the current develop, and merge conflicts were too many. I'm going try and redo it in a new branch with the latest changes

**Pull request closed without merging**